spec/features/node_merging_spec.rb

```diff
@@ -21,6 +21,8 @@
     click_link 'Merge'
   end
 
+  after { FileUtils.rm_rf(Dir.glob(Attachment.pwd + '*')) }
```

Comment:
Why is this change needed?

Reply:
In it's original location, it wouldn't trigger if the `expect` fails. Putting it in the `after` block ensures it triggers even if the test fails.

Reply:
it seems we'd have this "gotcha" with failing specs elsewhere in the specs/ folder, no?

Reply:
Looks like the issue is with the rails_helper and not this. I've removed this after callback and update the cleanup in the rails_helper